miden-lib/asm/miden/contracts/wallets/basic.masm

```diff
@@ -24,8 +24,12 @@ end
 #! Creates a note which sends the specified asset out of the current account
 #! to the specified recipient.
 #!
-#! Inputs: [ASSET, tag, aux, note_type, RECIPIENT, ...]
-#! Outputs: [note_idx, EMPTY_WORD, EMPTY_WORD, 0, 0, ...]
+#! Note: since this procedure must be only called, the operand stack should be padded to the 16 
+#! elements before being called. See the corresponding issue: 
+#! https://github.com/0xPolygonMiden/miden-base/issues/685
+#!
+#! Inputs: [ASSET, tag, aux, note_type, RECIPIENT, PAD(5)]
+#! Outputs: [note_idx, PAD(15)]
```

Comment:
Let's not change anything about `send_asset` procedure here. Otherwise, this would be breaking change.

Reply:
To clarify, we should keep the interface of this procedure as it was - but it is probably OK to modify its code.

Reply:
There will be breaking changes in any case because I changed the `tx::add_asset_to_note`, but I roll back the changes in the `send_asset` interface anyway.